compiler/vmgen.nim

```diff
@@ -1360,19 +1360,6 @@ proc genMarshalStore(c: PCtx, n: PNode, dest: var TDest) =
   c.gABx(n, opcMarshalStore, 0, c.genType(n[1].typ))
   c.freeTemp(tmp)
 
-const
```

Comment:
deadcode by maybe useful?

Reply:
This is very old (2013) and was used in genCopy but is long unused and outdated.